src/include/duckdb/main/client_config.hpp

```diff
@@ -115,6 +117,12 @@ struct ClientConfig {
 	//! Defaults to PhysicalMaterializedCollector
 	get_result_collector_t result_collector = nullptr;
 
+	//! If HTTP logging is enabled or not.
+	bool enable_http_logging = false;
```

Comment:
Shouldnt these be extension settings added by httpfs?

Reply:
Extension installs go through httplib before httpfs is even installed, so if we want to be able to log those, we can't add this setting in httpfs, that's too late